app/upgrades.go

```diff
@@ -25,16 +25,18 @@ import (
 )
 
 const (
-	TestnetUpgradeName = "v0.22.0-alpha.0"
+	MainnetUpgradeName = "v0.23.0"
+	TestnetUpgradeName = "v0.23.0-alpha.0"
 
+	MainnetStabilityCommitteeId = uint64(1)
 	TestnetStabilityCommitteeId = uint64(1)
```

Comment:
No need to change, but I think we should use the commitee keeper to look up the committee with the correct description (there doesn't seem to be a name field), and not hardcode to values in the future.